### PR TITLE
add default api version to swagger

### PR DESCRIPTION
**Pull request by sebader**
image

### CODE REVIEW

File: src/app/AlwaysOn.CatalogService/CatalogServiceHelpers.cs [hunk 1]
```diff
@@ -1,22 +1,21 @@
...
 using System.Net;
-using System.Text.Json;
 using System.Threading.Tasks;
 
 namespace AlwaysOn.CatalogService
 {
     public static class CatalogServiceHelpers
     {
+        public static int DefaultApiVersionMajor = 1;
+        public static int DefaultApiVersionMinor = 0;
```
Comment (msimecek) [inline src/app/AlwaysOn.CatalogService/CatalogServiceHelpers.cs:17]: Are you going to use this in the other places where version is referenced?

https://github.com/Azure/Mission-Critical-Online/blob/595748184a6c438ffe5482c326efe45b5598f3dc/src/app/AlwaysOn.CatalogService/Startup.cs#L78

https://github.com/Azure/Mission-Critical-Online/blob/595748184a6c438ffe5482c326efe45b5598f3dc/src/app/AlwaysOn.CatalogService/Startup.cs#L58

**Review by msimecek — CHANGES_REQUESTED** [review]

File: src/app/AlwaysOn.CatalogService/CatalogServiceHelpers.cs (hunk 1 above)
Comment (sebader) [reply src/app/AlwaysOn.CatalogService/CatalogServiceHelpers.cs:17]: yeah I was wondering: is the API definition version the same?

Comment (msimecek) [reply src/app/AlwaysOn.CatalogService/CatalogServiceHelpers.cs:17]: For the API definition that's more of a convenience thing - not necessary. But you hese new constants can be used for `o.DefaultApiVersion`, right?

Comment (sebader) [reply src/app/AlwaysOn.CatalogService/CatalogServiceHelpers.cs:17]: oh gotcha! I actually missed to port that part. Already did that in -Connected. fixed

**Review by msimecek — APPROVED** [review]

**Pull request merged**